.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question, please post it to [our Discourse site](https://discourse.julialang.org).
```

Comment:
Is it worth adding something like "If you have a question, or are just unsure if it is a bug, please post to our Discourse site"?